qa/L0_backend_python/python_test.py

```diff
@@ -297,7 +297,7 @@ def test_unicode(self):
         model_name = "string"
         shape = [1]
 
-        for i in range(3):
+        for i in range(2):
```

Comment:
I am wondering why are we reducing the iteration counts?

Reply:
I think the change of the iteration counts here is to match the responses from the model. The model was sending three responses for each dytpes, and now we remove the `np.object_` data type so we reduce the iteration counts to 2. @Tabrizian Would it make sense to add some comments here for this?

Reply:
I have added a comment.